schema/stacks/stack_schema_merge.go

```diff
@@ -163,6 +168,20 @@ func schemaForDependentComponentBlock(modMeta *tfmod.Meta) (*schema.BodySchema,
 		}
 		aSchema := tfschema.ModuleVarToAttribute(modVar)
 		aSchema.Constraint = tfschema.ConvertAttributeTypeToConstraint(varType)
+		aSchema.OriginForTarget = &schema.PathTarget{
+			Address: schema.Address{
+				schema.StaticStep{Name: "var"},
+				schema.AttrNameStep{},
+			},
+			Path: lang.Path{
+				Path:       modMeta.Path,
+				LanguageID: tfschema.ModuleLanguageID, // TODO: should this be StackLanguageID?
```

Comment:
```suggestion
				LanguageID: tfschema.ModuleLanguageID,
```
I think `ModuleLanguageID` is right – I test drove this a little and with `StackLanguageID` it wouldn't correctly connect the dots 😄